_posts/2017/2017-08-08-why-you-should-spend-time-on-your-translation-keys.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Why you should spend time on your translation keys
+date: "2017-08-08 09:34:02"
+image: blog-header/translation-keys.jpg
+categories: tech-stories
+authors: ["Christina"]
+---
+Translation keys are not important at all?
+Spending time on them is a waste of time?
+Let me convince you otherwise.
+
+We just finished the revision of our translation keys because we noticed the value of a good key structure.
+In the following, I will tell you why we changed the old system and what have been the most important aspects during this task.
```

Comment:
In the following, I will tell you why we looked at our translation keys more closely, and what have been the most important aspects.